ui-grid/ui-grid.d.ts

```diff
@@ -3766,6 +3766,14 @@ declare module uiGrid {
          * You may specify one of the sortingAlgorithms found in the rowSorter service.
          */
         sortCellFiltered?: boolean;
+        /**
+		* (optional) An array of sort directions, specifying the order that they should cycle through as 
```

Comment:
It seems like something that could be good for string literal types, once TypeScript 1.8 is released (see https://github.com/Microsoft/TypeScript/pull/5185)